inst/tests/tests.Rraw

```diff
@@ -1934,7 +1934,7 @@ l = list(data.table(a=1:2, b=7:8),
 test(676.1, rbindlist(l[1:3]), ans<-data.table(a=1:6,b=7:12), message="Column 2 [[]'baz'[]] of item 2 is missing in item 1.*Use fill=TRUE.*or use.names=FALSE")
 test(676.2, rbindlist(l[1:3], use.names=FALSE), ans)
 test(677.2, rbindlist(l[c(10,1,10,2,10)]), ans<-data.table(a=1:4,b=7:10), message="Column 2 [[]'baz'[]] of item 4 is missing in item 2")  # NULL items ignored
-test(677.2, rbindlist(l[c(10,1,10,2,10)], use.names=FALSE), ans)
+test(677.3, rbindlist(l[c(10,1,10,2,10)], use.names=FALSE), ans)
```

Comment:
Based on the above few lines, shouldn't this still be 677.2 and the one above 677.1 (i.e., these look like variations of 676.*)?

Reply:
Indeed! Thanks for the extra eyes.

Main goal of this PR is that if test `x` fails, you can <kbd>Ctrl</kbd> + <kbd>F</kbd> for `x` and find the right test, rather than to clean up all the tests (which would be a mammoth undertaking)